index.html

```diff
@@ -0,0 +1,243 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap');
+      </style>
+      <script
+      src="https://kit.fontawesome.com/675259b560.js" crossorigin="anonymous">
+    </script>
+    <title>Document</title>
+</head>
+<body>
+    <header>
+            <section id="home">
+                <nav id="navbar" class="open">
+                    <div class="container">
+                        <a href='#' class="logo">My Logo</a>
+                        <div class="boton"><i class="fa-solid fa-bars fa-2xs"></i></div>
+                    </div>
+                </nav>
+            </section>
+    </header>
+    <main>
+        <section class="portfolio">
+            <div>
+                <h1>Hey There. I'm Jonathan.</h1>
+            </div>
+            <div>  
+                <h2>I'm a Software Developer</h2>
+            </div>
+            <div>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        dont hesitate to contact me.</p>
+            </div>
+            
+            <div class="icons">
+                <a href="">
+                    <img src="Assets/Github.svg" alt="Github">
+                </a>
+                <a href="">
+                    <img src="Assets/Linkedin.svg" alt="Linkedin">
+                </a>
+                <a href="">
+                    <img src="Assets/Icon - Angelist.svg" alt="Angelist">
+                </a>
+                <a href="">
+                    <img src="Assets/Twitter.svg" alt="Twitter">
+                </a>
+                <a href="">
+                    <img src="Assets/Medium.svg" alt="Medium">
+                </a>
+            </div>
+        </section>
+        <section id="experience">
+            
+            <h2 class="title">My Recent Works</h2>        
+            <img class="vector" src="Assets/Vector.svg" alt="">        
+            <div>
+                <img  class="placeholder" src="Assets/Img Placeholder.svg" alt="">
+                
+                <div>
+                    <h2 class="work">Multi-Post Stories</h2>
```

Comment:
- [ ] Kindly use **h4** instead of **h2** because it optimizes the heading of your portfolio website and enhances the user experience. The `hx` tag is specifically designed for main headings or sub-headings, providing a clear and prominent representation of the page's primary topic or purpose. By utilizing the *h4* tag, you create a visual hierarchy that helps users quickly identify and determine the relevance of the content to their needs and interests. This improves the overall usability and navigation of your website, ensuring a more engaging and effective experience for visitors. Whereas you already have used the `h2` tag on **line no, 33** and the `h3` tag as I mentioned in my above *inline comment* so that's why please maintain a good hierarchy in your development for better accessibility and SEO.

- **Kindly for more information follow [this](https://seosherpa.com/header-tags/) article.**